symengine/tests/basic/test_arit.cpp

```diff
@@ -1376,6 +1376,10 @@ TEST_CASE("Expand2: arit", "[arit]")
     r1 = add(x, mul(i2, add(y, z)));
     r2 = expand(r1, false);
     REQUIRE(eq(*r1, *r2));
+
+    r1 = expand(pow(add(one, mul(i2, sqrt(add(y, z)))), i2));
+    r2 = expand(add(one, mul(i4, add(y, add(z, sqrt(add(y, z)))))));
+    REQUIRE(eq(*r1, *r2));
```

Comment:
I wonder if we could start using the `Expression` in tests, to make them more readable (possibly slower, but we don't care about that in tests).

Reply:
Yes, that's a good idea. That way, we can add to functionality in `Expression` if they are not added.